src/isa/ebpf/canonicalize.cc

```diff
@@ -1453,9 +1472,18 @@ void static_safety_check_win(inst * win_prog, int win_start, int win_end, prog_s
   }
   min_pkt_sz_inference_win(ss_win, pss_orig.static_state[win_start]);
 
+  int num_of_unsafe_ins = 0;
   for (int i = 0; i < win_len; i++) {
-    safety_chk_insn(win_prog[i + win_start], ss_win[i]);
+    try{
```

Comment:
Seems there is a space between `try` and `{` in google c++ style. Could you please format the code, i.e., modify it to `try {`?